### ISSUE TITLE
this does not work

### ISSUE THREAD

**Issue by XP-Fan** [lieuwex/whapp-irc#23]
After starting the docker immage if I try to establish an IRC-Connection to the port the connection gets immediately terminated.

**Comment by lieuwex** [comment]
Weird, does running the docker image with `-e "LOG_LEVEL=verbose"` tell anything useful in the logs?
Or maybe the raw IRC communication logs?

**Comment by chat-api** [comment]
Hi!
Have You seen latest version of Chat-API here?
its working fine without any issues.
https://chat-api.net
Thanks

**Comment by XP-Fan** [comment]
Honestly, I am absolutely new to linux and docker. As my username says I am an Windows XP-User and I just asked a friend of mine if he can set your bridge up for me, as it would be an extremely awesome help for me as a blind person if I could somehow use Whatsapp within my Miranda NG (multiprotocol messenger). So, I have no idea how I can start docker with logging or anything. I have a proxmox linux in my basement where we wanted to test this physically before we would have created a VBox VM to finaly integrate it within my existing Windows XP-Computer. If you don't mind, I can give you SSH-Access to the machine, and you can try it yourself.

**Comment by lieuwex** [comment]
Hey,

Sorry for taking a while to respond.
I really hope whapp-irc can be any use for you!

I could help you using SSH if you would like, but that would be without any liability on my part if anything goes wrong (just since everything feels a bit risky).
So maybe you feel more comfortable letting your friend do some setting up?

In any way I don't have any experience with proxmox linux, it looks like it has a web interface?
Is it possible to configure the docker containers from there?

**Comment by XP-Fan** [comment]
Hi,

At first thank you for responding anyway. Well, my friend is actually just a linux user, so he was able to install docker and start the image, and he tried to connect IRC which failed. He was actually the one who told me to contact you, so he is at the end of his knowledge. As this is a test machine, you can’t really destroy something. I just set up the computer for the only purpose of trying to get whapp-irc running there, as I wanted to know if my plan works before I’d go and install Virtualbox onto my XP-Machine. So, if you want, just feel free to experiment until you somehow get it working. There is nothing to be scared about as there is nothing else on the computer. As far as I know the the PVE Webinterface is only for management of virtual machines, but proxmox is based on debian, if that helps. However, if you feel an other linux would be better, then feel free to tell me which one and as long as it has a talking installer I can also switch it to something else.

**Comment by XP-Fan** [comment]
After I read on an orrifial proxmox documentation that they strongly discourage running docker images directly on the proxmox linux, I would now revert the test computer to Windows XP X86, (x64 does unfortunately not work at the moment because ACPI is not properly implemented in my CSM), then I would install VirtualBox, create a VM and then you could access it over VRDP. This way you can install whatever linux you like and get whapp-irc running for me. Then we also have to do it only once as Vbox VMs can be easily moved from one computer to another. However, please be aware that the VM can only have a maximum of 3,5 GB RAM then. Would this be okay?

**Comment by XP-Fan** [comment]
Well, Virtualbox is installed. If you link me to the ISO of your prefered OS I'll insert it into a VM and give you VRDP-Access

**Comment by lieuwex** [comment]
Nice, sounds good!
I don't think I necessarily really prefer one distro over another, as long
as it has a fairly normal base (so something debian related should be just
fine).

I don't really have experience with vrdp but I guess we will work something
out. :p

**Comment by XP-Fan** [comment]
Well, VRDP is basically pritty same as Microsoft RDP (Remote Desktop Protocol) and it is fully compatible to that. So if you have any sort of Windows PC you could simply press windows+r, enter "mstsc", press enter, and then the remotedesktop client opens up. There you'd just have to enter the adress I'd give you, edit settings if you want, (for example if you are on a slow network), and click connect. On Linux, there are implementations such as RDesktop or Freerdp, whereas RDesktop is recommended by VBox. However: If you are satisfied with a Debian, I can also give you SSH. Should it be Debian X64 or X86? Another alternative would be connecting directly to my host over AnyDesk. Just feel free to contact me privately as this way I can give you credentials.

**Comment by XP-Fan** [comment]
Oh sorry: I can offer you Skype, XMPP, IRC, Facebook, E-Mail or WhatsApp, whereas on WhatsApp, before whapp-irc is working, you need to expect either extremely slow responses or voice messages. At the moment I prefer Skype, Facebook, XMPP or IRC as they work seamlessly within Miranda NG, but just chose what you want as I am thankful to you for helping me anyway. Another alternative is also the telephone, of course.

**Comment by lieuwex** [comment]
I think IRC would be a good option, or otherwise Skype. But I'm afraid I'm a bit busy the next days. Maybe Friday is an option? Thursday I'm busy the whole day and Saturday I'm going away to the hazing of my fraternity.

**Comment by XP-Fan** [comment]
Okay, however in the evening like around 19:00 german time I might be ad dancing school. Well, tell me which IRC-Server you want me to connect to, or you can find me at skype as "live:b_peter", or as this sometimes not works you can add me over peter@neverseen.de<mailto:peter@neverseen.de> or b_peter@outlook.de<mailto:b_peter@outlook.de>

**Comment by XP-Fan** [comment]
Good morning! Could you please tell me about which time you would contact me?

**Comment by XP-Fan** [comment]
Are you here?

**Comment by XP-Fan** [comment]
I would still be available as Larissa did not want to go to dancing school with me.

**Comment by XP-Fan** [comment]
Well, I have set up a Debian x64 now. I hope that was the right decision. It would be ready for VRDP. If you prefer SSH I must somehow figure out how I can enable root access as it looks like this is not available by default.

**Comment by XP-Fan** [comment]
Do you think I could use an android VM instead of a real smartphone? Would a notebook-webcam be able to scan the code?

**Comment by XP-Fan** [comment]
hey? Are you here? More and more people are expecting me to be available on WhatsApp…

**Comment by XP-Fan** [comment]
Can you help me please?

**Comment by XP-Fan** [comment]
If Skype does not work for any odd reason just let me know and we'll find another solution.

**Comment by lieuwex** [comment]
Closing in favor of #28.
